### PR TITLE
Rust for ppc64le

### PR DESCRIPTION
**Pull request by mpe**
Here's a pull request with the ppc64le support. Now with CI wired up too.

### CODE REVIEW

**Comment by ojeda** [comment]
Oh, I was working on this in parallel on top of your patches. Let me try to compare...

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -103,6 +121,8 @@ jobs:
         run: sudo apt-get install -y qemu-system-x86
       - if: matrix.arch == 'arm64'
         run: sudo apt-get install -y qemu-system-arm
+      - if: matrix.arch == 'ppc64le'
+        run: sudo apt-get install -y qemu-system-ppc64
```
Comment (ojeda) [inline .github/workflows/ci.yaml:125]: `qemu-system-ppc` seems to be the package name, but it seems to install the same set.

**Comment by ojeda** [comment]
> Uses RUST_OPT_LEVEL_SIMILAR_AS_CHOSEN_FOR_C, not RUST_OPT_LEVEL_0 like

I quickly tested, and level 1 also fails.

**Comment by ojeda** [comment]
Since you took the time to send the PR with added CI, let's merge it and I can send my bits on top! :-)

**Review by ojeda — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (mpe) [reply .github/workflows/ci.yaml:125]: `qemu-system-ppc64` is a virtual package, I figure it's better to explicitly ask for the 64-bit version we need.

**Comment by mpe** [comment]
> Oh, I was working on this in parallel on top of your patches. Let me try to compare...

Oops sorry, I figured you were busy with other things. Thanks for merging it :+1:

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ojeda) [reply .github/workflows/ci.yaml:125]: Don't worry, it is just a nit: I was using the "real" package name in the other two arches, so I mentioned it. I typically look up in the repo index online the package names, and it does not seem to show the virtual ones there.

**Comment by ojeda** [comment]
No, thanks to you for the work! :-)

In case GitHub does not notify you on mentions: my bits are in #156, can you please take a look? Thanks in advance!

**Pull request merged**